proto/ord_schema.proto

```diff
@@ -523,34 +567,30 @@ message ReactionProduct {
   Compound compound = 1;
   bool is_desired_product = 2;
   Percentage compound_yield = 3;
-  ReactionAnalysis.AnalysisType yield_type = 4;
-  // Used to describe an UNKNOWN yield type.
-  string yield_type_custom = 5;
-  Percentage purity = 6;
-  enum SelectivityType {
-    SELECTIVITY_TYPE_UNKNOWN = 0;
-    EE = 1;  // Enantiomeric excess as a percentage.
-    ER = 2;  // Enantiomeric ratio. (x:1)
-    DE = 3;  // Diasteromeric ratio (x:1)
+  oneof kind {
+    ReactionAnalysis.AnalysisType type = 4;
+    string custom = 5;
   }
-  SelectivityType selectivity_type = 7;
-  string selectivity_type_custom = 8;
-  float selectivity_value = 9;
-  // TODO(kearnes): What does precision mean in this context?
-  float selectivity_precision = 10;
+  Percentage purity = 6;
   // TODO(ccoley): How to allow specification of the state of matter of the 
   // purified compound? For example, "___ was recovered as a white powder in
   // x% yield (y.z mg)". Or oils, crystal texture, etc. This is only relevant 
   // for compounds that are isolated.
-  string isolated_color = 12;
-  enum Texture {
-    TEXTURE_UNKNOWN = 0;
-    POWDER = 1;
-    CRYSTAL = 2;
-    OIL = 3;
+  // TODO(kearnes): Should this be an Observation message?
+  string isolated_color = 7;
+  message Texture {
+    enum TextureType {
+      UNSPECIFIED = 0;
+      POWDER = 1;
+      CRYSTAL = 2;
+      OIL = 3;
+    }
+    oneof kind {
+      TextureType type = 1;
+      string custom = 2;
+    }
   }
-  Texture isolated_texture = 13; 
-  string isolated_texture_custom = 14;
+  Texture texture = 8;
```

Comment:
Another case where it seems like we've increased the level of nesting to ```outcome.texture.type```

Reply:
Fixed in latest commit.